src/site/layouts/page.drupal.liquid

```diff
@@ -1,80 +1,84 @@
 {% include "src/site/includes/header.html" with drupalTags = true %}
+
 {% include "src/site/includes/alerts.drupal.liquid" %}
 {% include "src/site/includes/preview-edit.drupal.liquid" %}
 {% include "src/site/includes/breadcrumbs.drupal.liquid" %}
-<div id="content" class="interior">
-  <main class="va-l-detail-page">
-    <div class="usa-grid usa-grid-full">
 
-    {% include 'src/site/navigation/sidebar_nav.drupal.liquid' with sidebar %}
+<div itemscope itemtype="https://schema.org/FAQPage">
```

Comment:
I think it's okay to set the `itemtype="https://schema.org/FAQPage"` for all pages that use the `page` layout, because AFAIK all of those pages are the benefit hub pages like `/health-care/how-to-apply/` that are pretty much all FAQ pages conceptually.

Reply:
I'd say it fits about 80% of the benefits hub content (`page` content type). Some of the pages should have HowTo, but there's no great way to figure that out right now. This could be "editors choice" (like, provide a set of options to benefits hub editors, and populate schema markup accordingly) but that could be a later iteration.